src/javascript/app_2/pages/trading/components/elements/contracts_popup.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+
+class ContractsPopUp extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.handleVisibility   = this.handleVisibility.bind(this);
+        this.handleSelect       = this.handleSelect.bind(this);
+        this.setWrapperRef      = this.setWrapperRef.bind(this);
+        this.handleClickOutside = this.handleClickOutside.bind(this);
+        this.state = {
+            is_list_visible: false,
```

Comment:
should we use camelCase for variables name? for consistency 😅

Reply:
hi @kellybinary , for variable names we can use `_`.